enterprise/api/v1/views.py

```diff
@@ -27,6 +28,20 @@ class EnterpriseReadOnlyModelViewSet(viewsets.ReadOnlyModelViewSet):
     throttle_classes = (ServiceUserThrottle,)
 
 
+class EnterpriseReadOnlyModelViewSet(EnterpriseModelViewSet, viewsets.ReadOnlyModelViewSet):
+    """
+    Base class for all read only Enterprise model view sets.
+    """
+    pass
+
+
+class EnterpriseReadWriteModelViewSet(EnterpriseModelViewSet, viewsets.ModelViewSet):
```

Comment:
Deja vu!  Does this PR cover https://github.com/edx/edx-enterprise/pull/36#discussion-diff-98638742R30?